conf/messages.en

```diff
@@ -742,17 +742,17 @@ houseConsignment.combinedNomenclatureCode.NormalMode.error.required = Enter the
 houseConsignment.combinedNomenclatureCode.error.invalid = The combined nomenclature code must only include letters a to z and numbers 0 to 9
 houseConsignment.combinedNomenclatureCode.error.length = The combined nomenclature code must be 2 characters long
 
-
 houseConsignment.item.customsUnionAndStatisticsCode.CheckMode.title = What is the new Customs Union and Statistics (CUS) code for item {1} in house consignment {0}?
 houseConsignment.item.customsUnionAndStatisticsCode.NormalMode.title = What is the Customs Union and Statistics (CUS) code for item {1} in house consignment {0}?
 houseConsignment.item.customsUnionAndStatisticsCode.CheckMode.heading = What is the new Customs Union and Statistics (CUS) code for item {1} in house consignment {0}?
 houseConsignment.item.customsUnionAndStatisticsCode.NormalMode.heading = What is the Customs Union and Statistics (CUS) code for item {1} in house consignment {0}?
-houseConsignment.item.customsUnionAndStatisticsCode.hint = This will be 9 characters long and include both letters and numbers.
+houseConsignment.item.customsUnionAndStatisticsCode.hint = This will be 9 characters long and can include letters, numbers and a hyphen. For example, 1234567-1.
 houseConsignment.item.customsUnionAndStatisticsCode.CheckMode.error.required = Enter the new Customs Union and Statistics (CUS) code for item {1} in house consignment {0}
 houseConsignment.item.customsUnionAndStatisticsCode.NormalMode.error.required = Enter the Customs Union and Statistics (CUS) code for item {1} in house consignment {0}
+houseConsignment.item.customsUnionAndStatisticsCode.NormalMode.error.format = Enter the Customs Union and Statistics (CUS) code
```

Comment:
@kenneth-rayner do we need the NormalMode here?

Reply:
@jackheslop96  Don't even need that in either mode as the errors picked up are either empty, length, invalid characters or 'valid' code.

The valid code message is :

houseConsignment.item.customsUnionAndStatisticsCode.error.not.exists = Enter a valid Customs Union and Statistics (CUS) code

The unused is now removed